### PR TITLE
Cg styles albums

### PR DESCRIPTION
**Pull request by Cgatewood55**
equal sized images for albums page

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -115,7 +115,11 @@ p {
     width: 100%;
     height: auto;
 }
-
+/* ----albums styles----- */
```
Comment (ghost) [inline main.css]: This should go in the Albums section

File: main.js [hunk 2]
```diff
@@ -267,7 +267,7 @@ function albumsDomString(array){
     for(var i=0; i<array.length;i++){
         var albumsDetail = array[i];
         var albumsString = "";
-        albumsString += "<img src ='" + albumsDetail.image + "'/>";
+        albumsString += "<img class='albums-images'+ src ='" + albumsDetail.image + "'/>";
```
Comment (ghost) [inline main.js:270]: You should add a div to make styling easier

**Comment by ghost** [comment]
👍

**Comment by ledugani** [comment]
👍

**Pull request merged**